cmd/ctr/commands/images/pull.go

```diff
@@ -122,7 +126,9 @@ command. As part of this process, we do the following:
 			}
 			// TODO: Support unpack for all platforms..?
 			// Pass in a *?
-
+			if context.Bool("skip-metadata") && context.Bool("metadata-only") {
+				return fmt.Errorf("metadata-only and skip-metadata cannot be used together")
+			}
 			if context.Bool("metadata-only") {
 				sopts = append(sopts, image.WithAllMetadata)
 				// Any with an empty set is None
```

Comment:
Nothing needs to be done here to prevent unpack? I would have expected an unpack to fail trying to unpack the default platform.

Reply:
how about 
```
if context.Bool("metadata-only") == false && context.Bool("skip-metadata") == false {
return fmt.Errorf("metadata-only ==false and skip-metadata == false cannot be used together")
}
```
if  metadata-only ==skip-metadata== false 
```
if context.Bool("metadata-only") {
//step A
sopts = append(sopts, image.WithAllMetadata)
// Any with an empty set is None
// TODO: Specify way to specify not default platform
// config.PlatformMatcher = platforms.Any()
} else if !context.Bool("skip-metadata") {
//step B
sopts = append(sopts, image.WithAllMetadata)
}
```
step A will skip and step B will run . They run same func,This code looks odd.